packages/test/src/test-replay.ts

```diff
@@ -13,23 +13,25 @@ export interface Context {
 const test = anyTest as TestInterface<Context>;
 
 test.before(async (t) => {
+  // We don't want AVA to whine about unhandled rejections thrown by workflows
+  process.removeAllListeners('unhandledRejection');
   const logger = new DefaultLogger('DEBUG');
   // Use forwarded logging from core
-  const replayCore = await ReplayCore.install({ logger, telemetryOptions: { logForwardingLevel: 'DEBUG' } });
+  const replayCore = await ReplayCore.install({ logger, telemetryOptions: { logForwardingLevel: 'INFO' } });
```

Comment:
Do we need to configure the tracing filter too?

Reply:
Nah, no need. It's easy enough to just manually tweak when desired during test